src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -205,25 +202,28 @@ export class MatTabLink extends _MatTabLinkMixinBase
   }
 
   /**
-   * Ripple configuration for ripples that are launched on pointer down.
+   * Ripple configuration for ripples that are launched on pointer down. The ripple config
+   * is set to the global ripple options since we don't have any configurable options for
+   * the tab link ripples.
    * @docs-private
    */
-  rippleConfig: RippleConfig = {};
+  rippleConfig: RippleConfig & RippleGlobalOptions;
```

Comment:
Rather than doing this, shouldn't we make `RippleGlobalOptions` extend `RippleConfig`?

Reply:
I thought about that, but I don't think it would make sense that the global options extend the config of an individual ripple. Especially since `color`, `radius` etc. do not make sense in a global configuration.

This intersection type is just used to avoid having another getter that unwraps the global options, and to have type safety. I'm fine going with a getter though. Not feeling too strong about it (note that this property is internal-only)